docs/userguide.md

```diff
@@ -208,21 +210,28 @@ It is possible to disable propagation of a type that is configured for propagati
 `kubefed2` command:
 
 ```bash
-kubefed2 federate disable <FederatedTypeConfig name>
+kubefed2 disable <FederatedTypeConfig Name>
 ```
 
-This command will set the `propagationEnabled` field in the `FederatedTypeConfig` to `false`,
-which will prompt the sync controller for the type to be stopped.
+This command will set the `propagationEnabled` field in the `FederatedTypeConfig` 
+associated with this target API type to `false`, which will prompt the sync 
+controller to stop sync for this target API type.
```

Comment:
s/to stop sync for this target API type/for the target API type to be stopped/

I think it's important to note that the controller will be stopped, not just that propagation will be stopped.  The difference between the 2 statements has operational consequences.